openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -64,7 +64,7 @@ <h4 class="question__title ui-toggle-visibility__control">
                 <textarea
                     id="assessment__rubric__question--feedback__value"
                     placeholder="{{ rubric_feedback_default_text }}"
-                    maxlength="500"
+                    maxlength="100000"
```

Comment:
If the database imposes no _practical_ limit and this is only really enforced client-side, I'm not too strongly against a huge value here.

_Should_ someone leave a comment that large? Probably not.
Should we prevent them from doing so if they _really_ want to? Eh...

If the database doesn't care, I don't either.